c/tls.c

```diff
@@ -76,21 +76,20 @@ static int isTLSV13Available(TlsSettings *settings) {
 #define TLS_MIN_DEFAULT TLS_V1_2
 #define TLS_MAX_DEFAULT TLS_V1_3
 
-static char *TLS_NAMES[5] = {
+#define TLS_NAMES_COUNT 5
+static char *TLS_NAMES[TLS_NAMES_COUNT] = {
   "invalid",
   "TLSv1.0",
   "TLSv1.1",
   "TLSv1.2",
   "TLSv1.3"
 };
 
-#define TLS_NAMES_LENGTH 5
-
 static int getTlsMax(TlsSettings *settings) {
   if (settings->maxTls != NULL) {
-    for (int i = 0; i < TLS_NAMES_LENGTH; i++) {
+    for (int i = 0; i < TLS_NAMES_COUNT; i++) {
```

Comment:
Nit: you could do `sizeof(TLS_NAMES) / sizeof(TLS_NAMES[0])` (we should use a macro for this in fact because this is ugly), so that you have a dependency on the thing you want the size of instead of a dependency on a dependency which can change (e.g., someone starts using something else for the size of `TLS_NAMES` and doesn't check if `TLS_NAMES_COUNT` is used somewhere else). Of course, my proposal can also break if someone starts `malloc`ing `TLS_NAMES`, but there seem to be solutions for that too :-)